README.md

```diff
@@ -1,2 +1,42 @@
 # go-btfs-common
 Common packages among various go-btfs projects
+
+## File Structure
+
+```
+.
+├── LICENSE
+├── Makefile
+├── README.md
+├── go.mod
+├── info
+│   └── node.go (deprecating)
+├── protos
+│   └── node
```

Comment:
We talked in the same protos folder for each module, not one folder for escrow etc. Is there any change?

Reply:
After Friday's sync I think we already agreed on one folder for each module since each module will be sealed up in its own package, and each package is required to be an individual folder.

Reply:
ok, we now follow this rule. thanks.

Reply:
Good to know